libs/ui/src/components/NeoSelect/NeoSelect.vue

```diff
@@ -1,36 +1,47 @@
 <template>
-  <o-field :label="label">
-    <o-select :placeholder="placeholder">
-      <option
-        v-for="(option, index) in options"
-        :key="index"
-        :value="option.value">
-        {{ option.text }}
-      </option>
-    </o-select>
-  </o-field>
+  <NeoField :message="description" class="neo-select">
+    <o-field :label="label">
+      <o-select
+        :placeholder="placeholder"
+        :class="{ 'with-label': hasLabel }"
+        icon-right="chevron-down">
+        <option
+          v-for="(option, index) in options"
+          :key="index"
+          :value="option.value">
+          {{ option.text }}
+        </option>
+      </o-select>
+    </o-field>
+  </NeoField>
 </template>
 
 <script lang="ts" setup>
```

Comment:
we'll be missing default `rootClasses` and `selectClasses` check out #6197

Reply:
I am not sure if this is right decision to be honest... We are trying to migrate to Nuxt 3. In Nuxt 3 composition API is preferred type of components because it is known issue that Options API and mixins are breaking typescript in Vue... Issue that you are referring mentions some new Neo components and both of them were created using Options API and I believe we should refactor it to composition.

Reply:
`libs/ui` is based on Oruga Vue2 version, without this class we can't expand/overwritte component

Reply:
Hmm okay then